modules/common-transformer-stream/src/main/scala/com/snowplowanalytics/snowplow/rdbloader/transformer/stream/common/Resources.scala

```diff
@@ -131,7 +132,7 @@ object Resources {
     }
   }
 
-  private def mkResolver[F[_]: Sync: InitSchemaCache: InitListCache](resolverConfig: ResolverConfig): Resource[F, Resolver[F]] =
+  private def mkResolver[F[_]: Sync: CreateResolverCache](resolverConfig: ResolverConfig): Resource[F, Resolver[F]] =
```

Comment:
There's nothing wrong with this.  I'll just point out that the following also compiles:

```scala
private def mkResolver[F[_]: Async](resolverConfig: ResolverConfig): Resource[F, Resolver[F]] =
```

I don't know which is better, I just thought it was interesting.